web/docs/auth/migrate-from-11.md

```diff
@@ -0,0 +1,318 @@
+
+```wasp title="main.wasp"
+app myApp {
+  wasp: {
+    version: "^0.11.0"
+  },
+  title: "My App",
+  auth: {
+    userEntity: User,
+    externalAuthEntity: SocialLogin,
+    methods: {
+      gitHub: {}
+    },
+    onAuthFailedRedirectTo: "/login"
+  },
+}
+
+entity User {=psl
+  id                        Int           @id @default(autoincrement())
+  username                  String        @unique
+  password                  String
+  externalAuthAssociations  SocialLogin[]
+psl=}
+
+entity SocialLogin {=psl
+  id          Int       @id @default(autoincrement())
+  provider    String
+  providerId  String
+  user        User      @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId      Int
+  createdAt   DateTime  @default(now())
+  @@unique([provider, providerId, userId])
+psl=}
+```
+
+### New Auth Models
+
+From 0.12.X onwards, authentication is based on the auth models which are automatically set up by Wasp. You don't need to take care of the auth fields anymore.
+
+The `User` model is now just a business logic model and you use it for storing the data that is relevant for your app.
+
+```wasp title="main.wasp"
+app myApp {
+  wasp: {
+    version: "^0.12.0"
+  },
+  title: "My App",
+  auth: {
+    userEntity: User,
+    methods: {
+      gitHub: {}
+    },
+    onAuthFailedRedirectTo: "/login"
+  },
+}
+
+entity User {=psl
+  id Int @id @default(autoincrement())
+psl=}
+```
+
+You can read more about the new auth system in the [Auth Entities](./entities) section.
+
+## How to Migrate?
+
+Migrating your existing app to the new auth system is a two-step process:
+1. Migrate to the new auth system
+1. Cleanup the old auth system
+
+:::caution Migrating a deployed app
+
+To migrate a deployed app, you need to go through the migration steps while deploying the app in between the steps.
+
+Read more about migrating a deployed app in the [Migrating a Deployed App](#migrating-a-deployed-app) section.
+:::
+
+### 1. Migrate to the New Auth System
+
+If you have an existing app with authentication set up, you can follow these steps to migrate to the new auth system:
+
+1. Upgrade Wasp to the latest version
+1. Bump the version to `^0.12.0` in `main.wasp`
+1. Create the new auth tables in the database by running:
+
+  ```bash
+  wasp db migrate-dev
+  ```
+
+1. Create a data migration script in `src/server/migrateToNewAuth.ts`
+
+    - Below you can find [examples of migration scripts](#example-migration-scripts) for each of the auth methods
+
+1. Add the migration script to the `db.seeds` config:
+  ```wasp title="main.wasp"
+  app myApp {
+    wasp: {
+      version: "^0.12.0"
+    },
+    // ...
+    db: {
+      seeds: [
+        import { migrateEmailAuth } from "@server/migrateToNewAuth.js",
+        import { migrateGoogleAuth } from "@server/migrateToNewAuth.js",
+      ]
+    },
+  }
+  ```
+1. Run the migration script by running:
+  ```bash
+  wasp db seed
+  ```
+1. Verify still works as expected
```

Comment:
Could we provide a bit more information here? How should they verify it? What exactly, auth in general? I would give a bit more guidance. Also a dot at the end of the sentence.